_layouts/base.html

```diff
@@ -2,6 +2,14 @@
 <html lang="en">
 <head>
 	{% include head.html %}
+
+	<script>
+		if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+			document.documentElement.classList.add('dark-mode');
+		} else {
+			document.documentElement.classList.remove('dark-mode');
+		}
+	</script>	
```

Comment:
Light/Dark toggle with an icon will be in a separate PR.
![image](https://github.com/defold/defold.github.io/assets/7230306/cf0a7d80-259c-45ab-954e-c1ee51f7c86a)
![image](https://github.com/defold/defold.github.io/assets/7230306/7f80fdda-0b5c-400b-904a-aa20049b3f05)

Reply:
> Light/Dark toggle with an icon will be in a separate PR. ![image](https://private-user-images.githubusercontent.com/7230306/268509527-cf0a7d80-259c-45ab-954e-c1ee51f7c86a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NjQyNDksIm5iZiI6MTcxMDk2Mzk0OSwicGF0aCI6Ii83MjMwMzA2LzI2ODUwOTUyNy1jZjBhN2Q4MC0yNTljLTQ1YWItOTU0ZS1jMWVlNTFmN2M4NmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjBUMTk0NTQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWY1ZGI3MmVjNTQ0NjhjMmE5ZGFkZmFmOGMyNDg3NzBjYzA3NTZiYTAyODIzZGJmMTViNTZlYzE1MTEyZjkxNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.x1xu-jcx7_Avt2fz_TpiTxXlCm5OR3Qvs8ucmtf38tw) ![image](https://private-user-images.githubusercontent.com/7230306/268509537-7f80fdda-0b5c-400b-904a-aa20049b3f05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NjQyNDksIm5iZiI6MTcxMDk2Mzk0OSwicGF0aCI6Ii83MjMwMzA2LzI2ODUwOTUzNy03ZjgwZmRkYS0wYjVjLTQwMGItOTA0YS1hYTIwMDQ5YjNmMDUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjBUMTk0NTQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTNkOTAxN2NkODg4YTQ3NjQyOWI3ZmFkYjRjMmQ3OTJiYjVmZTAwOTE5OWNhMGU2NDAyOGIwYzNiZmEwN2IzNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.qEGh2yfVs-XEiS3ooBgYN5SWbh_v6muXMzIwUl9Hplw)

Hi @mikatuo
Thank you for your work on adding the dark mode. Are you still planning to add a toggle?

I also noticed a small bug. When scrolling horizontally across a block of code, there is no darkened background beyond the initial borders.
<img width="768" alt="sc" src="https://github.com/defold/defold.github.io/assets/70640955/201cb2ce-9c7c-44cd-9092-adec7f2d312c">

Reply:
> Thank you for your work on adding the dark mode. Are you still planning to add a toggle?

Thanks! 😄 
Not anymore. It uses dark/light theme based on system preferences which should be enough for 99% of users.

> I also noticed a small bug.

Would you mind creating a new issue for this bug? I could create one but I don't know on what page was it. There are a few different pages with code snippets.

Reply:
> Would you mind creating a new issue for this bug? I could create one but I don't know on what page was it. There are a few different pages with code snippets.

This appears in code snippets where lines of code extend beyond the initial scope and there is horizontal scrolling. Not sure I explained it clearly :) You can take a look at this page for example:
https://defold.com/manuals/atlas/#creating-a-texture-resource-at-runtime